app/jest/my-dashboard/my-projects/assignUsersToRoles.test.tsx

```diff
@@ -50,36 +54,30 @@ jest.mock('services/bceid-webservice', () => ({
 }));
 
 describe('assign user to roles tab', () => {
-  it('Should display correct property options from the drop-down box', () => {
-    render(<UserRoles selectedRequest={{ ...sampleRequest, environments: ['dev', 'test'], devIdps: ['idir'] }} />);
-    expect(screen.getByRole('option', { name: 'Dev' }));
-    expect(screen.getByRole('option', { name: 'Test' }));
-    expect(screen.getByRole('option', { name: 'IDIR' }));
-    expect(screen.getByRole('option', { name: 'Email' }));
-  });
-
-  it('Should be able to input some keywords in the Search Criteria input field', () => {
-    render(<UserRoles selectedRequest={{ ...sampleRequest }} />);
-    const searchInput = screen.getByRole('textbox');
-    fireEvent.change(searchInput, { target: { value: 'sample_input' } });
-    expect(screen.getByDisplayValue('sample_input')).toBeInTheDocument();
-  });
-
   it('Should be able to list search results, after click on the Search button; Should be able to click the View button for listed search result, and corresponding modal showing up', async () => {
     render(<UserRoles selectedRequest={{ ...sampleRequest, environments: ['dev', 'test'], devIdps: ['idir'] }} />);
-    const searchUserInput = screen.getByRole('textbox');
-    fireEvent.change(await searchUserInput, { target: { value: 'sample_user' } });
+    await waitFor(async () => {
+      fireEvent.click(await screen.findByRole('option', { name: 'Dev' }));
```

Comment:
@kurokn you need to validate all three environments and IDPs. Each IDP has different options. Example: when you select BCeID, you can choose `Display Name` or `Username`, which you need to validate

Reply:
@NithinKuruba i followed the same path as what you did in the `assignServiceAccountsToRoles.test` and trying to update the dropdown selection value, but seems the fireEvent.click on the `option idp` is not working properly, even the statement passed
current version only renders the component with one specific idp situation only, for checking the associated user-selection-prop